compute/compute/README.md

```diff
@@ -0,0 +1,53 @@
+# Code samples for the Compute Engine library
```

Comment:
Should the directory rather be something like `compute/client_library/` instead of `compute/compute`? The latter is hard to dictate what kind of samples live under the directory by its name.

Reply:
That's a great suggestion. Applying it now.